### ISSUE TITLE
ZeroDivisionError: division by zero

### ISSUE THREAD

**Issue by turovapolina** [pymzml/pymzML#215]
Dear all,

I am using package pymzml but one problem appeared after reinstallation of windows system on my laptop.
I am trying to open mzml file with this commands:

run = pymzml.run.Reader("D:/asp/extracts/Exp1 mzml/s8/" + filename)

But unfortunately I receive this error:
![image](https://user-images.githubusercontent.com/61549119/77825822-d7193d80-711c-11ea-8720-29f8a63c69cb.png)

Do you have any ideas why it can happen? It worked perfectly before.
 
Best regards,
Polina Turova

**Comment by fu** [comment]
Hi Polina,

do you build_index_from_scratch? Could you share you mzML with us?

Cheers

.c

**Comment by turovapolina** [comment]
> Hi Polina,
> 
> do you build_index_from_scratch? Could you share you mzML with us?
> 
> Cheers
> 
> .c

Dear Christian,

Thank you very much for your reply!
This is really weird but problem dissapeared by itself when I tryed to parse mzml file on the other day. Probably I just spontaneously chose another version which was suitable for my files. I already send to you my mail example of my mzml file and description of my errors.
Thank you! And I am sorry that I disturbed you!

Best regards,
Polina
